engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -19,34 +19,40 @@
 import org.terasology.rendering.opengl.BaseFBOsManager;
 import org.terasology.rendering.opengl.FBOManagerSubscriber;
 import com.google.common.base.Objects;
-import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
 
 import static org.lwjgl.opengl.EXTFramebufferObject.GL_FRAMEBUFFER_EXT;
 import static org.lwjgl.opengl.EXTFramebufferObject.glBindFramebufferEXT;
 
 /**
- * TODO: Add javadocs
+ * Binds the given FBO to the GL_FRAMEBUFFER target, making it the read framebuffer target as well as
+ * the draw framebuffer target.
+ *
+ * This is useful for operations such as off-screen rendering.
```

Comment:
It isn't in the scope of this PR, so a separate PR would make sense. That been said, it is probably a single line change in the code and perhaps some rewriting of the javadocs, which needs to be done anyway. So, it wouldn't be horribly out of place in this PR as its own commit. But I'm not fussed either way.